src/storage/storage.cc

```diff
@@ -180,6 +180,29 @@ rocksdb::Options Storage::InitRocksDBOptions() {
 }
 
 Status Storage::SetOptionForAllColumnFamilies(const std::string &key, const std::string &value) {
+  if ("compression" == key) {
```

Comment:
I think we can take this into a single function like `SetCompressionOption` since it's a special setting option.

Reply:
done